code/Metric_visualization.R

```diff
@@ -0,0 +1,66 @@
+## Visualization of model evaluation metrics.
+
+library(tidyverse)
+
+stream_type <- readRDS('data/stream_type .RDS')
+stream_type$seg_id_nat <- as.character(stream_type$seg_id_nat)
+
+dot_size <- rgnc_dat_filter %>%
```

Comment:
This is okay for now because we're not doing anything formal (so no need to make changes), but usually we want to make each script reproducible. Currently, `rgcn_dat_filter` and `rmse` and `nse_metric` can't be reproduced from this script. If I wanted to run these locally, I'd have to find the script that produces thoses dataframes.